deploy/runtime/quickstart/PuertoRicoMiniGrid.lk

```diff
@@ -0,0 +1,331 @@
+// Puerto Rico Mini Grid macro using PuertoRicoMiniGrid.sam
```

Comment:
Nate had requested we change the name of this (including the file name, I think) since it could be used for any mini grid, not just mini grids in Puerto Rico